scripts/lib.sh

```diff
@@ -192,7 +195,21 @@ create_summary_file()
 	local agent="${AGENT_DEST}"
 	[ "$AGENT_INIT" = yes ] && agent="${init}"
 
-	local -r agent_version=$("$agent" --version|awk '{print $NF}')
+	local agent_version
+	if [ "${RUST_AGENT}" == "no" ]; then
+		agent_version=$("$agent" --version|awk '{print $NF}')
+	else
+		local -r agentdir="${RUST_SRC_PATH}/$(basename ${RUST_AGENT_PKG} .git)/src/agent"
+		agent_version=$(cat ${agentdir}/VERSION)
+	fi
+
+	local REAL_AGENT_PKG
+
+	if [ "$RUST_AGENT" == "no" ]; then
+		REAL_AGENT_PKG=${GO_AGENT_PKG}
```

Comment:
this indent (on my github screen) does not quite align with the one above - possibly extra ` ` space in there somewhere?

Reply:
:) Is there some better way to take care of indent. I cannot use expandtab in vim, since some indent is tab, but the other is whitespace.. Maybe we can change all indent to whitespace.. It will be easier for us to handle indent

Reply:
Strange, they are two tabs in my vim.. I don't known why it misindent in github screen..